tests/CI/test_weight_only_gpu.py

```diff
@@ -0,0 +1,260 @@
+import os
+import torch
+import unittest
+import shutil
+from intel_extension_for_transformers.transformers.modeling import AutoModelForCausalLM
+from intel_extension_for_transformers.transformers import WeightOnlyQuantConfig
+from transformers import AutoTokenizer
+from intel_extension_for_transformers.utils.utils import get_gpu_family, _ipex_available
+import torch.utils.data as data
+from torch.utils.data import DataLoader
+import torch.nn.functional as F
+
+
+if _ipex_available:
+    gpu_name = get_gpu_family()
+
+
+def dequantize(weight, scales, zero_points, group_size, gemm_num=1):
+    k = weight.size()[-2]
+    n = weight.size()[-1]
+    weight = weight.reshape([gemm_num, k, n])
+    n = n * 2
+    group_num = int(k / group_size)
+    scales = scales.reshape([gemm_num, group_num, n])
+    zero_points = zero_points.reshape([gemm_num, group_num, int(n / 2)])
```

Comment:
just a reminder: we convert s4_sym weight to int8, and then convert s4_asym zp to int8, abvious that weight & zp is not match.
The ut does not throw an error because we select sym-quantization, kernel will not care the zp value.

Reply:
This function is invalid in this test case, so I had removed.